src/views/layouts/main.handlebars

```diff
@@ -1,22 +1,30 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en-us">
+
 <head>
-	<meta charset="UTF-8">
 
-        <!-- CSS -->
-		<link rel="stylesheet" href="/assets/css/devdash.css" type="text/css" />
+    <meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<!-- Bulma -->
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.2/css/bulma.min.css">
+    <script defer src="https://use.fontawesome.com/releases/v5.0.0/js/all.js"></script>
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="../public/css/devdash2.css" type="text/css" />
+
+    <!-- Jquery -->
+    <script src="https://code.jquery.com/jquery.js"></script>
 
-        <!-- Jquery --> 
-		<script src="https://code.jquery.com/jquery.js"></script>
+    <!-- Javascript -->
+    <script src="../public/js/devdash.js"></script>
```

Comment:
Does this path work? The server uses the 'public' directory for static files, so I would think this should just be /js/devdash.js